### PR TITLE
Greg/helper tests

### PR DESCRIPTION
**Pull request by GregTheGreek**
Knocked out tests for:
- slotToEpoch()
- getEpochStartSlot()
- isActiveValidator() 

Also added a folder `tests/utils` that has a file `validator.ts` that contains helper functions `generateValidators()` and `generateValidator()`

### CODE REVIEW

File: beaconChain/tests/helpers/stateTransitionHelpers.test.ts [hunk 1]
```diff
@@ -230,3 +233,79 @@ describe("readUIntBE", () => {
...
+    {test: 1, expected: 0},
+    {test: 10, expected: 0},
+    {test: 100, expected: 1},
+    {test: 1000, expected: 15},
+    {test: 10000, expected: 156},
+    {test: 100000, expected: 1562},
+    {test: 1000000, expected: 15625},
+  ];
+  for (const pair of pairs) {
+    it(`Return ${pair.expected} a value of ${pair.test}`, () => {
```
Comment (ansermino) [inline beaconChain/tests/helpers/stateTransitionHelpers.test.ts]: Peculiar phrasing, consider `slot <test> should map to epoch <expected>`

Comment (ansermino) [reply beaconChain/tests/helpers/stateTransitionHelpers.test.ts]: Same goes for tests that follow

File: beaconChain/tests/utils/validator.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {Validator} from "../../types";
+
+export function generateValidator(activation: number = null, exit: number = null): Validator {
```
Comment (wemeetagain) [inline beaconChain/tests/utils/validator.ts]: Might be nice to add a small comment sharing the intent of this function.

File: beaconChain/tests/helpers/stateTransitionHelpers.test.ts [hunk 3]
```diff
@@ -230,3 +233,79 @@ describe("readUIntBE", () => {
...
+      assert.equal(result, pair.expected);
+    });
+  }
+});
+
+describe("isActiveValidator", () => {
+  it("should be active", () => {
+    const v: Validator = generateValidator(0, 100);
+    const result: boolean = isActiveValidator(v, 0);
+    assert.equal(result, true);
```
Comment (ansermino) [inline beaconChain/tests/helpers/stateTransitionHelpers.test.ts]: Convention tells me assert.isTrue would be more descriptive

**Review by wemeetagain — APPROVED** [review]

File: beaconChain/tests/utils/validator.ts (hunk 2 above)
Comment (ansermino) [inline beaconChain/tests/utils/validator.ts]: params should be optional

**Comment by GregTheGreek** [comment]
@ansermino updated.

**Review by ansermino — APPROVED** [review]

**Pull request merged**